refurb/checks/datetime/unreliable_utc_usage.py

```diff
@@ -0,0 +1,65 @@
+from dataclasses import dataclass
+from typing import Final
+
+from mypy.nodes import CallExpr, MemberExpr, NameExpr
+
+from refurb.error import Error
+from refurb.settings import Settings
+
+
+@dataclass
+class ErrorInfo(Error):
+    """
+    Because naive `datetime` objects are treated by many `datetime` methods
+    as local times, it is preferred to use aware datetimes to represent times
+    in UTC.
+
+    This check affects `datetime.utcnow` and `datetime.utcfromtimestamp`.
+
+    Bad:
+
+    ```
+    from datetime import datetime
+
+    now = datetime.utcnow()
+    past_date = datetime.utcfromtimestamp(some_timestamp)
+    ```
+
+    Good:
+
+    ```
+    from datetime import datetime, timezone
+
+    datetime.now(timezone.utc)
+    datetime.fromtimestamp(some_timestamp, tz=timezone.utc)
+    ```
+    """
+
+    name = "unreliable-utc-usage"
+    code = 176
+    categories = ("datetime",)
+
+
+_replacements: Final = {
+    "utcnow": ("()", "now(tz=timezone.utc)"),
+    "utcfromtimestamp": ("(...)", "fromtimestamp(..., tz=timezone.utc)"),
+}
+
+
+def check(node: CallExpr, errors: list[Error], settings: Settings) -> None:
```

Comment:
The `settings` argument can be removed here, since Refurb will automatically inject the settings if the `settings` arg is present. Make sure to remove the `Settings` import as well.